### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by biosbeta** [SamsungLabs/RGBD-FGN#1]
Excuse me, can the graspnet-1Billion Dataset be directly used for your network model? Dataset

**Comment by Andyyoung0507** [comment]
@biosbeta 
Will directly using the cross entropy loss function to predict collisions lead to low prediction accuracy due to imbalanced sample categories? I have this problem. Thank you!
